src/pages/SearchDetails/components/CategorySelect/index.tsx

```diff
@@ -0,0 +1,39 @@
+import cn from 'utils/ts/classNames';
+
+import styles from './CategorySelect.module.scss';
+
+interface Props {
+  className: string;
+  onClick: () => void;
+  isCafe: boolean;
+}
+
+export default function CategorySelect({ className, isCafe, onClick }: Props) {
```

Comment:
통상적으로 두개의 상태로 관리되는 ui 는 토글이라는 말을 더 많이 쓰는것같아요
```suggestion
export default function CategoryToggle({ className, isCafe, onClick }: Props) {
```

Reply:
저도 이름을 정하는 데 고민이 많이 되더라구요. 원래 CategoryToggleButton이나 CategoryToggle로 정하고 만들고 있는데 읽고 생각할 때 어감이 좀 이상했어요. 그리고 on/off의 상태가 아니기 때문에 toggle이 무조건 맞다는 생각이 안 들더라구요. 이건 다른 분들한테도 물어보고 싶어요.

Reply:
토글은 a <=> !a 의 느낌이 강한데
카페와 음식점이라는 서로 다른 값을 선택하는 느낌이라 select도 좋은 것 같아요